Description: Do not remove docs/index.html
 Commit 9ddda31cd3a208411c642d8c081207718782f80c changed ./configure
 which now doesn't generate the docs/index.html file, but forgot to
 modify the 'distclean' rule to not remove this file.
Author: Ilias Tsitsimpis <iliastsi@debian.org>

Index: b/ghc.mk
===================================================================
--- a/ghc.mk
+++ b/ghc.mk
@@ -1460,7 +1460,6 @@ distclean : clean
 	$(call removeFiles,docs/users_guide/ug-book.xml)
 	$(call removeFiles,docs/users_guide/ug-ent.xml)
 	$(call removeFiles,docs/users_guide/ghc_config.py)
-	$(call removeFiles,docs/index.html)
 	$(call removeFiles,libraries/prologue.txt)
 	$(call removeFiles,distrib/configure.ac)
 	$(call removeFiles,ch01.html ch02.html index.html)
